awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx

```diff
@@ -84,10 +84,15 @@ function ToolbarDeleteButton({
     }
   }, [isKebabified, isModalOpen, onKebabModalChange]);
 
+  function verifyUsername(item) {
+    return item.username;
+  }
+
   const renderTooltip = () => {
+    const criteria = itemsToDelete.every(verifyUsername) ? 'username' : 'name';
     const itemsUnableToDelete = itemsToDelete
       .filter(cannotDelete)
-      .map(item => item.name)
+      .map(item => item[criteria])
```

Comment:
why not just:

.map(item => item.username || item.name) ?

Reply:
@jlmitch5, updater per your suggestion. Thanks.